spec/openapi.yaml

```diff
@@ -6115,21 +6146,27 @@ components:
         page:
           type: number
           description: The current page of results
+          example: 3
         page_size:
           type: number
           description: The amount of results returned per page
+          example: 50
         total_count:
           type: number
           description: The total number of results in the results set
+          example: 80
         page_count:
           type: number
           description: The total number of pages in the results set
+          example: 5
         next_uri:
           type: string
           description: Link to the next page of results
+          example: example.url
```

Comment:
This pagination looks like it is used in v1 as v2 doesn't provide next or previous page links.

Reply:
Good catch - theres a new v2 pagination object which I've now created and implemented here!